lib/active_record_extended/query_methods/either.rb

```diff
@@ -12,14 +10,14 @@ def either_join(initial_association, fallback_association)
         associations        = [initial_association, fallback_association]
         association_options = xor_field_options_for_associations(associations)
         condition__query    = xor_field_sql(association_options) + "= #{table_name}.#{primary_key}"
-        outer_joins(associations).where(Arel.sql(condition__query))
+        left_outer_joins(associations).where(Arel.sql(condition__query))
```

Comment:
[ar_outer_joins](https://github.com/varvet/ar_outer_joins) still works fine, but it's been unnecessary since since https://github.com/rails/rails/pull/12071 added `left_outer_joins` in Rails 5.0 https://github.com/rails/rails/blob/5-0-stable/guides/source/5_0_release_notes.md They both use the same `Arel::Nodes::OuterJoin`